survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
+  EC2_INSTANCE_TAG = "${var.env}-RabbitMQ"
```

Comment:
Just need to set an appropriate tag and declare this module again with settings appropriate for jenkins